toolbox/cluster/set_scale.sh

```diff
@@ -36,18 +35,30 @@ fi
 source ${THIS_DIR}/../_common.sh
 
 INSTANCE_TYPE=${1}
-ANSIBLE_OPTS="${ANSIBLE_OPTS} -e machineset_instance_type=${INSTANCE_TYPE}"
-
+shift 
+SCALE=${1}
 shift
+if [ $# -ne 0 ]; then
+    if [ "${1}" == "--base-machineset" ]; then
+        shift
+        BASE_MACHINESET=${1}
+        ANSIBLE_OPTS="${ANSIBLE_OPTS} -e base_machineset=${BASE_MACHINESET}"
+        shift
+        if [ $# -ne 0 && "${1}" == "--force" ]; then
+            ANSIBLE_OPTS="${ANSIBLE_OPTS} -e force_scale=true"
+        fi
+    elif ["${1}" == "--force" ]; then
+        echo "Setting cluster ${INSTANCE_TYPE} machinesets to have scale '${SCALE}'"
+        ANSIBLE_OPTS="${ANSIBLE_OPTS} -e force_scale=true"
+    fi
```

Comment:
here you can simplify a bit by removing the duplicated `if` block:
```
# pseudo bash
if $1 == --base-machineset; then
       shift
       BASE_MACHINESET=${1}
       ANSIBLE_OPTS=...
       shift
fi

if ${1:-} == --force; then
    echo ... 
    ....
fi
```

not the bash notation `${1:-}` which gives the value of `$1` *if it is set, otherwise the empty default value. 
General form is `${var:-default_value}` which `$var` or `default_value`.
Very useful when `set -o nounset` (`set -u`) is defined!